test/test_runlib.py

```diff
@@ -216,6 +217,75 @@ def test_exclude_subsubdir(self):
     self.assertListEqual(sorted(artifacts_dict.keys()),
         sorted(["foo", "bar", "subdir/foosub1", "subdir/foosub2"]))
 
+class TestInTotoRecordRun(unittest.TestCase):
+  """"Test in_toto_run(name, material_list, product_list,
+  link_cmd_args, key=False, record_byproducts=False). """
```

Comment:
lol, I do the same thing :P

The way I see it, we could either remove them, or list them and say what a "valid" range of each parameter would look like (I think the former is less of a hassle tho)